lib/validators/movieget.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
+
+  title: Joi.string().min(1).max(255).optional(),
+
+  year: Joi.number().integer().min(1878).max(9999).optional(),
+
+  from_year: Joi.number().integer().min(1878).max(9999).optional()
+  .when('year', { is: Joi.exist(), then: Joi.forbidden() }),
```

Comment:
You may be able to get away with using an [`oxor`](https://hapi.dev/family/joi/?v=16.0.1#objectoxorpeers-options) here for both of these (though you may need to update Joi to be able to use `oxor`).